docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -289,7 +289,7 @@ Options:
 
 - Interval: (Interval or cron required) Evergreen will create a version and run the specified tasks every X hours, with
     X being specified by this field. Unless set to a future time, the first run will happen immediately after the settings are saved.
-- Cron: (Interval or cron required) Evergreen will create a version on the specified cron schedule.
+- Cron: (Interval or cron required) Evergreen will create a version on the specified [cron schedule](https://crontab.guru/).
```

Comment:
Does this use the same cron-parsing logic as `cron`? If so, can we make the docs more detailed and/or copy-paste [the cron docs](https://github.com/evergreen-ci/evergreen/blob/94d9e76fe8c3ff742603f9bbde62d06fe9a690d4/docs/Project-Configuration/Project-Configuration-Files.md?plain=1#L277-L283)?

Reply:
updated